Source/Core/DolphinWX/MainNoGUI.cpp

```diff
@@ -380,7 +380,7 @@ static Platform* GetPlatform()
 
 int main(int argc, char* argv[])
 {
-  std::string boot_filename;
+  std::string boot_filename, user_directory;
```

Comment:
This variable should be moved nearest to its use. In this case, above the if statement down below.